lib/config.js

```diff
@@ -57,6 +57,26 @@ class Config {
     }
   }
 
+  get defaults() {
```

Comment:
This isn't quite functionality equivalent. The accessor returns a new POJO per invocation, and disallows overrides.

```js
class A {
  get b() { return {} }
}

a = new A
> A {}
a.b = 1
> 1
a.b
> {}
```

I think, this me need to be returned to the prototype.

Reply:
Hmmmm good point. Will fix. I've been using `lebab` to convert, and i think it may have messed something up.